FETCH_ERROR=Falha ao salvar log de atividade ou requisitar dados da API externa